src/main/java/org/dita/dost/reader/GenListModuleReader.java

```diff
@@ -396,7 +396,9 @@ public void startElement(final String uri, final String localName, final String
 
         if (foreignLevel > 0) {
             foreignLevel++;
-            return;
+            if (!DitaClass.isValidDitaClass(classValue)) {
```

Comment:
Missed this (comment was made on an old commit but only showed up today) -- renaming now.